src/datasets-md5.c

```diff
@@ -50,12 +51,13 @@ bool Md5StrCompare(void *a, void *b)
 uint32_t Md5StrHash(void *s)
 {
     const Md5Type *str = s;
-    uint32_t hash = 5381;
-
-    for (int i = 0; i < (int)sizeof(str->md5); i++) {
-        hash = ((hash << 5) + hash) + str->md5[i]; /* hash * 33 + c */
-    }
-    return hash;
+#if defined(TEST_HASH_MURMUR)
```

Comment:
You went from an inlined hash calculation, to a non-inlined hash calculation through the function call.
This is not necessarily bad, it's just different.
If you absolutely must have the same behaviour as the previous code, you could make HashFunctionDjb2() an inline function and hope the compiler inlines it.